public/javascripts/gformFileUpload.js

```diff
@@ -27,22 +27,32 @@
     var submitButton = form.find(".govuk-button--secondary")
     var formComponentId = input.attr("id");
     var uploadedFiles = $("#" + formComponentId + "-files")
-    var frm = $("#gf-form");
     submitButton.css("display", "")
     submitButton.on("click", function(evt) {
       formGroup.hide()
       uploadedFiles.empty().append(startProgressBar());
       submitButton.css("display", "none")
       evt.preventDefault();
 
+      var gfForm = $("#gf-form");
       $.ajax({
-          type: frm.attr('method'),
-          url: frm.attr('action'),
-          data: frm.serialize()
-      }).then(function (){
-          frm.submit();
+          type: gfForm.attr("method"),
+          url: gfForm.attr("action"),
+          data: gfForm.serialize()
+      }).done(function (){
+          var upscanForm = $("#gf-upscan-"+formComponentId);
```

Comment:
This is `upscan` specific, so we cannot do that. but it doesn't matter since this will go away anyway, see my previous comments.